Rakefile

```diff
@@ -7,6 +7,8 @@ require 'rubygems'
 require 'rake/testtask'
 require 'yard'
 require "#{File.dirname(__FILE__)}/lib/tasks/all.rb"
+require_relative 'lib/tasks/helpers/prompt'
```

Comment:
This `Rakefile` file lives at `<PROJECT ROOT>/lib/tasks`. So the prompt file's location relative to `Rakefile's` location should be simply `helpers/prompt`. You could also clean up the `all.rb` file if you wanted to:

```diff
- require "#{File.dirname(__FILE__)}/lib/tasks/all.rb"
+ require_relative 'all'
- require_relative 'lib/tasks/helpers/prompt'
+ require_relative 'helpers/prompt'
```

Reply:
I'm seeing `Rakefile` and `all` living at the project root?

Reply:
Whoops! I got this one mixed up with the .rake files that were edited. The path does indeed need to be different here given that `Rakefile` is up 2 directory levels from the others. Sorry for the mixup.

Reply:
I guess `all` isn't project root whoops but in any case I'll leave this be!